### ISSUE TITLE
Checking Sanger encoding fails when found J

### ISSUE THREAD

**Issue by magicDGS** [magicDGS/ReadTools#21]
When Sanger-like enconding (Phred+33) comes from Illumina 1.8+, it allows the value 41(J). Nevertheless, my checker is using 40 as the highest value for this Phred score.

Change in the next release.

**Comment by magicDGS** [comment]
In addition, the thrown error is not meaningful:

```
DEBUG    2016-04-18 11:52:20    Main    DEBUG mode on
INFO    2016-04-18 11:52:20    TrimFastq    ReadTools v.0.2.0.r_0c18ac39
DEBUG    2016-04-18 11:52:20    TrimFastq    Running on Mac OS X 10.8.5 (x86_64)
DEBUG    2016-04-18 11:52:20    TrimFastq    ERR657750_1.fastq
INFO    2016-04-18 11:52:20    TrimFastq    Running TrimFastq with arguments: --input1 ERR657750_1.fastq --input2 ERR657750_2.fastq --output ERR657750_jartrimmed
INFO    2016-04-18 11:52:20    TrimFastq    Output will be in Sanger format independently of the input format
DEBUG    2016-04-18 11:52:29    QualityEncodingDetector    Read 1000001 records from [FastqReader[ERR657750_1.fastq Line:4000005]].
DEBUG    2016-04-18 11:52:29    QualityEncodingDetector    Read 1000001 records from [FastqReader[ERR657750_1.fastq Line:4000005]].
DEBUG    2016-04-18 11:52:38    QualityEncodingDetector    Read 1000001 records from [FastqReader[ERR657750_2.fastq Line:4000005]].
DEBUG    2016-04-18 11:52:38    QualityEncodingDetector    Read 1000001 records from [FastqReader[ERR657750_2.fastq Line:4000005]].
DEBUG    2016-04-18 11:52:38    FastqReaderPairedSanger    Encoding for the original FASTQ reader: Standard
DEBUG    2016-04-18 11:52:38    TrimFastq    Running paired end
DEBUG    2016-04-18 11:52:38    TrimFastq   
org.vetmeduni.utils.fastq.QualityUtils$QualityException: Found 74(J) in Sanger encoded base
    at org.vetmeduni.utils.fastq.QualityUtils.checkEncoding(QualityUtils.java:215)
    at org.vetmeduni.utils.fastq.QualityUtils.checkEncoding(QualityUtils.java:242)
    at org.vetmeduni.utils.fastq.StandardizerAndChecker.checkMisencoded(StandardizerAndChecker.java:125)
    at org.vetmeduni.utils.fastq.StandardizerAndChecker.checkMisencoded(StandardizerAndChecker.java:78)
    at org.vetmeduni.utils.fastq.StandardizerAndChecker.standardize(StandardizerAndChecker.java:142)
    at org.vetmeduni.utils.fastq.StandardizerAndChecker.standardize(StandardizerAndChecker.java:169)
    at org.vetmeduni.io.readers.fastq.paired.FastqReaderPairedSanger.next(FastqReaderPairedSanger.java:59)
    at org.vetmeduni.io.readers.fastq.paired.FastqReaderPairedSanger.next(FastqReaderPairedSanger.java:37)
    at org.vetmeduni.tools.implemented.TrimFastq.processPE(TrimFastq.java:175)
    at org.vetmeduni.tools.implemented.TrimFastq.process(TrimFastq.java:148)
    at org.vetmeduni.tools.implemented.TrimFastq.runThrowingExceptions(TrimFastq.java:125)
    at org.vetmeduni.tools.AbstractTool.run(AbstractTool.java:58)
    at org.vetmeduni.readtools.Main.main(Main.java:68)
ERROR    2016-04-18 11:52:38    TrimFastq    Found 74(J) in Sanger encoded base
ERROR    2016-04-18 11:52:38    Main    Unexpected error. Please contact with Daniel Gomez-Sanchez (daniel.gomez.sanchez@hotmail.es)
```

**Comment by magicDGS** [comment]
Added bug fix in 092eaaf746bad1c97d244d39581b62de102e8d36 and better logging in 5b23810d5723d4c3610588d7a4c10c691f2ff0bb.
